types/node/globals.d.ts

```diff
@@ -360,7 +360,7 @@ declare const Buffer: {
      * Creates a new Buffer using the passed {data}
      * @param data data to create a new Buffer
      */
-    from(data: any[]): Buffer;
+    from(data: number[]): Buffer;
```

Comment:
seems the variant `Buffer.from(object[, offsetOrEncoding[, length]])` is missing which was partly supported by this one, see https://nodejs.org/dist/latest-v11.x/docs/api/buffer.html#buffer_class_method_buffer_from_object_offsetorencoding_length

Reply:
Symbols are tricky, I'm not touching this one, feel free to add it in another PR.
Please note this will likely only work in the `3.2` folder.